src/WinRPM.jl

```diff
@@ -50,20 +50,24 @@ if is_unix()
         unsafe_string(x.body), x.http_code
     end
 elseif is_windows()
+   function download(url::AbstractString, filename::AbstractString)
+        ps = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+        tls12 = "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
+        client = "New-Object System.Net.Webclient"
+        # in the following we escape ' with '' (see https://ss64.com/ps/syntax-esc.html)
+        downloadfile = "($client).DownloadFile('$(replace(url, "'", "''"))', '$(replace(filename, "'", "''"))')"
+        run(`$ps -NoProfile -Command "$tls12; $downloadfile"`)
+        filename
+    end
+
     function download(source::AbstractString; retry=5)
-        dest = Vector{UInt16}(261)
+        filename = joinpath(@__FILE__, "..", "..", "cache", basename(source))
```

Comment:
there should be a DownloadString method instead of using this cache folder for things we aren't currently putting there - downloading anything that happened to be called "index" or ".gitignore" would be bad